.gitignore

```diff
@@ -1,3 +1,3 @@
 .DS_Store
 **node_modules/**
-.vscode
\ No newline at end of file
+.vscode
```

Comment:
note: this is just tracking an addition and subtraction of a single line `package-lock.json`.  You can ignore this file as no change is being made here, effectively.